test/reducers/entities/posts.test.js

```diff
@@ -39,6 +39,32 @@ describe('Reducers.posts', () => {
                 },
             });
         });
+
+        it('should not add postId to postsInChannel', () => {
```

Comment:
Could you clarify a bit on the expected behaviour here since we only expect this when `postsInChannel` has no entry for the channel yet?
```suggestion
        it('should not add postId to postsInChannel when postsInChannel[channelId] is undefined', () => {
```

It might also be nice to have a test for when `postsInChannel` is defined as well

Reply:
Changed it to be descriptive. 

>It might also be nice to have a test for when postsInChannel is defined as well

There is a test case below to cover. Can you check if that is what you had in mind?